snmp-observ-lib/panels/fleet.libsonnet

```diff
@@ -0,0 +1,129 @@
+local g = import '../g.libsonnet';
+local commonlib = import 'common-lib/common/main.libsonnet';
+local xtd = import 'github.com/jsonnet-libs/xtd/main.libsonnet';
+
+{
+  new(signals, this):: {
+    deviceTable:
+      local fieldOverride = g.panel.table.fieldOverride;
+      signals.fleetInterface.interfacesCount.asTable(name='Network devices', format='table', filterable=true)
+      + g.panel.table.panelOptions.withDescription('Network devices overview.')
+      + signals.system.sysName.asTableColumn(format='table')
+      + signals.system.version.asTableColumn(format='table')
+      + signals.system.uptime.asTableColumn(format='table')
+      + commonlib.panels.system.table.uptime.stylizeByName('Uptime')
+      + signals.cpu.cpuUsage.asTableColumn(format='table')
+      + commonlib.panels.generic.table.percentage.stylizeByName('CPU utilization')
+      + signals.memory.memoryUsage.asTableColumn(format='table')
+      + commonlib.panels.generic.table.percentage.stylizeByName('Memory utilization')
+      + signals.fleetInterface.networkOutBitPerSec.asTableColumn(format='table')
+      + commonlib.panels.generic.table.coldHotGauge.stylizeByName('Transmitted')
+      + signals.fleetInterface.networkInBitPerSec.asTableColumn(format='table')
+      + commonlib.panels.generic.table.coldHotGauge.stylizeByName('Received')
+      + signals.fleetInterface.networkInErrorsPerSec.asTableColumn(format='table')
+      + signals.fleetInterface.networkOutErrorsPerSec.asTableColumn(format='table')
+      + signals.fleetInterface.networkInDroppedPerSec.asTableColumn(format='table')
+      + signals.fleetInterface.networkOutDroppedPerSec.asTableColumn(format='table')
+      + signals.fleetInterface.ifInUnknownProtos.asTableColumn(format='table')
+      + signals.alerts.alertsWarning
+        .withExprWrappersMixin(['count by (%s)(' % std.join(',', this.config.instanceLabels + this.config.groupLabels), ')'])
+        .asTableColumn(format='table')
+      + signals.alerts.alertsCritical
+        .withExprWrappersMixin(['count by (%s)(' % std.join(',', this.config.instanceLabels + this.config.groupLabels), ')'])
+        .asTableColumn(format='table')
+      + {
```

Comment:
This is a very wide table (wider than the dashboard, in the screenshot, as it cuts off the "dropped in" column)

I noticed the job column is all the same, maybe we want to do a rewrite or collapse by default?

Reply:
let me try something..

Reply:
I noticed the change to use `networkAllErrorsPerSec`, could you show a screenshot of what that looks like?

Reply:
yepp, see below

Reply:
Awesome thank you! That looks much much better